### PR TITLE
URL search parameters-based navigation

### PR DESCRIPTION
**Pull request by kostysh**

### CODE REVIEW

File: src/dc/components/accommodation-blocks/hotel-venue-distance.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const HotelVenueDistance = ({hotelLatitude, hotelLongitude}) => {
         return (<></>);
     }
 
-    let venuePoint = new Geopoint(Number(venueConfig.destinationLocation.latitude), Number(venueConfig.destinationLocation.longitude));
+    let venuePoint = new Geopoint(Number(venueConfig.destinationCity.latitude), Number(venueConfig.destinationCity.longitude));
```
Comment (mtahon) [inline src/dc/components/accommodation-blocks/hotel-venue-distance.js:13]: Don't agree with this change, the location of the venue is more precise than the city. I used the exact location of the event in this variable.

File: src/dc/components/venue-context/theme-context.js [hunk 2]
```diff
@@ -11,10 +11,9 @@ const getDate = (strDate, defaultValue) =>{
 }
 
 export const venueConfig={
-    originIata:venue.originIata,
-    destinationIata:venue.destinationIata,
-    originLocation:venue.originLocation,
-    destinationLocation:venue.destinationLocation,
```
Comment (mtahon) [inline src/dc/components/venue-context/theme-context.js:17]: Please keep a reference to the exact venue within the city, this is **Agora Bogotá Convention Center.**

**Review by mtahon — CHANGES_REQUESTED** [review]
Hi @kostysh , I did not reviewed all logic but a no-go for me is the location of the venue is being removed and replace by city coordinates.

File: src/dc/components/venue-context/theme-context.js (hunk 2 above)
Comment (kostysh) [reply src/dc/components/venue-context/theme-context.js:17]: venueConfig usage is slightly changed. for the navigation purposes, I have extended this configuration.
Now not only IATA code is configured for the origin and destination but the whole airport object.
```
originAirport: venue.originAirport,
destinationAirport: venue.destinationAirport,
destinationLocation: venue.destinationLocation,
venueName: venue.venueName
```

File: src/dc/components/accommodation-blocks/hotel-venue-distance.js (hunk 1 above)
Comment (kostysh) [reply src/dc/components/accommodation-blocks/hotel-venue-distance.js]: fixed

**Pull request merged**